test/e2e/multiple_policies_for_same_node.go

```diff
@@ -0,0 +1,96 @@
+package e2e
```

Comment:
Please see https://github.com/nmstate/kubernetes-nmstate/pull/351, add _test suffix to the file name

Reply:
This was not resolved.